service_identity/src/main/kotlin/org/thechance/service_identity/domain/entity/UserOptions.kt

```diff
@@ -0,0 +1,9 @@
+package org.thechance.service_identity.domain.entity
+
+data class UserOptions(
+    val page: Int,
+    val limit: Int,
+    val query: String?,
+    val permissions: List<Int>?,
+    val country: List<String>?
+)
```

Comment:
why query is null ?

Reply:
@Saif-Alhaider if no query it works as a get function else search.